workshop/content/advanced/resource-types/example-in_Python/_index.md

```diff
@@ -0,0 +1,302 @@
+---
+title: 'Example in Python'
+date: 2021-11-16T20:55:12Z
+weight: 320
+---
+
+### Overview
+
+In this module, you will follow steps to register an existing, example private extension written in Python with the AWS CloudFormation registry in your AWS account. You will also navigate through the example source code implementation for the resource type to understand key concepts of the resource type development workflow.
+
+
+### Topics Covered
+
+By the end of this lab, you will be able to:
+
+* understand key concepts to leverage when you develop a resource type;
+* use the [CloudFormation Command Line Interface (CLI)](https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/what-is-cloudformation-cli.html) to create a new project, run contract tests, and submit the resource type to the CloudFormation registry in your AWS account;
+* understand how to use the [AWS SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) to manually test your resource type handlers.
+
+
+
+### Start Lab
+
+In this lab, you will go on a deep dive for the code of an existing, example resource type: you will explore steps and a number of considerations to make when modeling and implementing a resource type. For information on how to get started with the CloudFormation CLI to create a new project, see [Walkthrough: Develop a resource type](https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/resource-type-walkthrough.html).
+
+
+#### Example resource type walkthrough
+
+Let's dive deeper into key considerations you should make when you design the model schema of your resource type. As an example, you will use an existing, sample resource type called [AWSSamples::EC2::ImportKeyPair](https://github.com/aws-cloudformation/aws-cloudformation-samples/tree/main/resource-types/awssamples-ec2-importkeypair/python), that illustrates an example of importing and managing an imported [Amazon EC2](https://aws.amazon.com/ec2/) key pair with CloudFormation.
```

Comment:
Given the example repo is outside of the workshop repo, there's a risk that it could move or change. Also, given we provide sample code in other parts of this workshop under the `code` directory, it might be best to vendor this example resource type there too.